### DISCUSSION TITLE
Sign up for GitHub’s Customer Research Panel!

### DISCUSSION THREAD

**Discussion by tuves** [community/events#11] [Announcements]
Once you sign up you’ll have a chance to win a coveted neon Octocat sign 😎

If you end up participating in research, you’ll have the opportunity to give feedback on early designs, get access to betas, and inform our product direction! Plus, we’ll compensate you for your time 🤑

[Click here to sign up!](https://github.co/GH_Research_Panel)

![IMG_7737](https://user-images.githubusercontent.com/2140740/199746104-7b91232e-f8a4-44e1-8281-29d542f14ef6.jpg)

For more information about participating in GitHub Research studies, [click here](https://cxr.github.com/).

### Sweepstakes Terms & Conditions

After signing up for the Research Panel at https://github.co/GH_Research_Panel, you will be entered into a sweepstakes to be conducted on November 30, 2022, for one limited edition neon Octocat sign. 

The drawing will be conducted by the GitHub Customer Research Team in San Francisco, CA. The chance of winning a prize is approximately 1 in 300. The winner will be notified immediately by email and be asked for a mailing address to send the prize.

NO PURCHASE NECESSARY. Open only to legal residents of the 50 U.S. and D.C., must be 18 or older. Sweepstakes ends November 30, 2022. For Official Rules, see https://github.co/Research-Panel-Sweepstakes.

**Comment by osmandarra** [comment]
I need an advanced sample of northwind in serenity frame work.

**Reply by samus-aran** [reply]
Hi @osmandarra 👋 

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [serenity docs](https://serenity.is/docs/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Nzizah12** [reply]
Thanks

**Comment by osmandarra** [comment]
Can you please help me ?

**Comment by xTalhaAhmedCho** [comment]
![chrome_eTjRY3DcEQ](https://github.com/user-attachments/assets/247865c8-f382-48c5-a43a-de6bd50d3803)

`Dear GitHub Support Team,`

> I am writing to request the re-enablement of GitHub Actions for my account, **@TalhaAhmedCho**. I have noticed that GitHub Actions is currently disabled for my account. 

I understand that my GitHub Actions may have been disabled due to `misunderstanding of limits`. If my actions inadvertently violated any GitHub terms or policies, I sincerely apologize. It was not my intention to misuse the platform. I am committed to using GitHub Actions responsibly and in accordance with all guidelines in the future.

> I value GitHub's services greatly, especially GitHub Actions, which is an **_integral part of my workflow_** for personal project. I rely on GitHub Actions for `learning new technologies. `

**_I kindly request you to review my account and re-enable GitHub Actions. I am willing to cooperate fully and take any necessary steps to ensure I comply with GitHub's policies and guidelines in the future._**

`My GitHub username is:`  **@TalhaAhmedCho**

Thank you for your **_time and consideration_**. I look forward to your positive response and the re-enablement of GitHub Actions.

Sincerely,

Talha Ahmed Chowdhury
[Linkedin](https://www.linkedin.com/in/TalhaAhmedCho/) | [Facebook](https://www.facebook.com/@TalhaAhmedCho) | [YouTube](https://www.youtube.com/@TalhaAhmedCho) | [Twitter](https://x.com/_TalhaAhmedCho)
